### ISSUE TITLE
Installation Failure on iOS 13.3.1

### ISSUE THREAD

**Issue by taronaeo** [pixelomer/AltDeploy#69]
Installation File: Spotify++ v8.5.7_v1.0
Error: An error occurred while installing the app.
Console.app report:
```
Error installing app. 3892346881 (ApplicationVerificationFailed). Failed to verify code signature of /private/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.1oWXaT/extracted/ppsideloader.app : 0xe8008001 (An unknown error has occurred.)
```

iOS Version: 13.3.1
MacOS Version: Catalina 10.15.2

Application previously installed on iOS 13.3 weeks ago worked. Now it doesn't.

**Comment by AfricanGigolo** [comment]
the same problem!!!

**Comment by Qyntc** [comment]
Altdeploy doesn't support 13.3.1 yet.

You need to downgrade iOS to 13.3 or lower

**Comment by taronaeo** [comment]
iOS 13.3 or 13.3.1 doesn't matter. Before updating my iOS, I was on 13 and
it didn't work either (previously it did work).

**Comment by sapcons** [comment]
Same here - after upgrading to 13.3.1 I was force to reinstall the app. 

Writing to device: OK


Error installing app. 3892346881 (ApplicationVerificationFailed). Failed to verify code signature of /private/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.YE .......

**Comment by telepati** [comment]
++ apps never worked. But many apps working. You must try with a different app.

**Comment by taronaeo** [comment]
Problem doesn't reside in the Spotify IPA. I've tried PopcornTimeiOS as
well and verified that the problem is residing in AltDeploy only.

Method 1:
Try installing Spotify++ and PopcornTimeiOS.
Result? Both failed with the error described above.

Method 2:
Try using NullXImpactor
Result? Spotify++ and PopcornTimeiOS installed perfectly well.

**Comment by telepati** [comment]
@taronaeo I tried many apps with NullXImpactor it never installed. These impactors always have this type of problem even CydiaImpactor. But try this trick;  turn .ipa to zip and extract it. zip Payload folder again and change zip to ipa. Try to install this new ipa. Many apps working with this trick. 

Also if NullXImpactor working for you use it, forget to AltDeploy.

**Comment by mightymercado** [comment]
@telepati Could you give clearer instructions? So I should turn .ipa into zip, extract it but what do you mean by zipping the Payload folder? Do I remove it and zip? Thank you in advance.

**Comment by telepati** [comment]
@mightymercado change extensions of .ipa to .zip. Then extract it this zip. When you extract it you will get the Payload folder. After extract to zip don't touch anything just re-zip this Payload folder. You don't need to change the name. Then turn this Payload.zip to Payload.ipa. And install this payload.ipa with AltDeploy or NullXImpactor.

**Comment by taronaeo** [comment]
@telepati Good workaround, it worked! But I'm still leaving this issue open for now for further investigations as I believe non-technical users should not be given these set of instructions to "patch" their IPA files, but instead done automatically.

**Comment by JakubZebrowski** [comment]
It didn’t work for me, I’m trying to instal ispoofer for Pokemon go and the issue is the same. I’ve followed the instructions, changed the ipa to zip, extracted that zip, compressed the folder and then changed the zip to ipa...

**Comment by zeroecho** [comment]
I have two iPhones, one on 13.3 and the other on 13.3.1. The same IPA works on 13.3 but fails with the error on 13.3.1. I tried the extract IPA and re-zip approach. It doesn't work.

**Comment by JakubZebrowski** [comment]
The only thing that helped me was to downgrade iOS back to 13.3

**Comment by morphymore** [comment]
Same issue here with 13.3.1 (same IPA worked on 13.3 but not on 13.3.1).

And sadly, rolling back to 13.3 is no longer possible since Apple has now stopped signing 13.3. Guess we'll just have to sit and wait for an update now.

**Comment by WeiqiangYu0528** [comment]
my iphone version is 13.3, and it does not working either. Every time it crashed in the beginning installation.

**Comment by jamwal** [comment]
Is there any plan to fix this issue ? 
We can side load apps on latest devices because of this  :(

**Comment by pixelomer** [comment]
If an IPA worked on iOS 13.3 but it doesn't work on iOS 13.3.1, then it's probably because the IPA contains dynamic libraries. iOS 13.3.1 has a bug that prevents people with free developer accounts from installing applications with dynamic libraries in them.

**Comment by telepati** [comment]
> If an IPA worked on iOS 13.3 but it doesn't work on iOS 13.3.1, then it's probably because the IPA contains dynamic libraries. iOS 13.3.1 has a bug that prevents people with free developer accounts from installing applications with dynamic libraries in them.

Yes, we know that issue but I can install the same .ipa with NullxImpactor is it that means something wrong in AltDeploy?

**Comment by evansgo** [comment]
Any movement on this one? It doesn’t seem to work on 13.4 either 
Thanks
